aip/general/0213.md

```diff
@@ -18,24 +18,105 @@ into taking major version changes or even lead to dependency conflicts.
 
 However, there are also cases where common structures are valuable, especially
 where a concept is well-known and it is sufficiently clear that it will not
-change. Common protos serve this use case.
+change. Common protos serve this use case. There are two forms of common protos:
+organization-specific and global.
 
-## Guidance
+## Guidance for API protos
 
-The public representation of APIs **should** be self-contained, meaning that
+The public representation of APIs **must** be self-contained, meaning that
 all protos used by the API originate in the same proto `package`, except for
 common protos, which **may** be used freely in any API.
 
 APIs **must not** define a set of API-specific common protos which live outside
-of its versioning structure. This prevents independent movement of particular
-versions and also causes problems for client libraries in many languages that
-compile the proto messages into classes.
+their versioning structure. (This is sometimes attempted as a way of reusing
+messages across multiple versions. The complexities introduced are definitely
+not worth the benefits.)
 
-APIs **should not** directly depend on protos defined in other APIs. Instead,
+APIs **must not** directly depend on protos defined in other APIs. Instead,
```

Comment:
nit: use a bulleted structure similar to other AIPs?

Reply:
Whoops, yes. Not sure why I moved away from that. I haven't updated this PR with that change yet, as it might mess up some other comments. But I'll do it when we've agreed other changes.